src/10_functions.py

```diff
@@ -2,11 +2,17 @@
 
 # YOUR CODE HERE
 
+
+def checkEven(num):
+    if num % 2 == 0:
+        print("Even")
+    else:
+        print("Odd")
+
+
```

Comment:
Exactly! Here, we can save 3 lines of code by simply writing `return n % 2 == 0`  which will return a Boolean in it of itself. Honestly though, sometimes I find the way you did it to be more readable. However I thought I'd mention it so you are aware if you were not already!